tools/shared.py

```diff
@@ -827,6 +827,11 @@ def check_engine(engine):
     print 'Checking JS engine %s failed. Check %s. Details: %s' % (str(engine), EM_CONFIG, str(e))
     return False
 
+def make_command(filename, engine=None, *args):
```

Comment:
how about `make_js_command`? (not in jsrun, just here and places using this? inside jsrun it's clear the context is js)